tests/glvis_driver.py

```diff
@@ -0,0 +1,144 @@
+import argparse
+import sys
+import os
+from skimage.io import imread
+from skimage.metrics import peak_signal_noise_ratio
+
+test_cases = {
+    "magnify": "*****",
+    "axes1": "a",
+    "axes2": "aa",
+    "mesh1": "m",
+    "mesh2": "mm",
+    "cut_plane": "i",
+    "cut_plane_rotate": "iyyyy",
+    "cut_plane_rotate_back": "iyyyyYYYY",
+    "cut_plane_transl": "izzzz",
+    "cut_plane_transl_back": "izzzzZZZZ",
+    "orient2d_1": "R",
+    "orient2d_2": "RR",
+    "orient2d_3": "RRR",
+    "orient2d_4": "RRRR",
+    "orient2d_5": "RRRRR",
+    "orient2d_6": "RRRRRR",
+    "orient3d": "Rr",
+    "perspective": "j",
+}
```

Comment:
Can you clarify how these are used?

Reply:
Those were originally going to be used as input to the `-keys` command-line argument to test operations on raw meshes, but since we're only testing stream files in this PR, we could probably remove that for now.

Reply:
Or you can keep it but comment that it is not used currently

Reply:
Done